food_delivery/foodapp/serializers.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+from .models import *
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        exclude = ['password']
+
+
+class RestaurantSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Restaurant
+        fields = '__all__'
+
+
+class MenuItemSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MenuItem
+        fields = '__all__'
+
+
+class CustomerSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Customer
+        fields = '__all__'
+
+
+class DeliveryPersonSerializer(serializers.ModelSerializer):
```

Comment:
add some custom create or update. plain serializers are simple. you need to show some differences in serializers. if all serializers are same, then it looks like you just added just for sake of completion.